op-e2e/actions/proofs/isthmus_setcode_tx_test.go

```diff
@@ -0,0 +1,142 @@
+package proofs_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/core/vm/program"
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/require"
+
+	batcherFlags "github.com/ethereum-optimism/optimism/op-batcher/flags"
+	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
+)
+
+func TestSetCodeTxTypeIsthmus(gt *testing.T) {
```

Comment:
It would be better to follow the pattern to run this test with a test matrix that covers multiple cases of valid/junk claim. See `simple_program_test.go` for an example.

Reply:
Added!